src/app.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import configureStore from "./store/configureStore";
+import AppRouter from "./routers/AppRouter";
+// import Filters from "./components/Filters";
+// import EventItem from "./components/EventItem";
+import { addEvent, editEvent, removeEvent } from "./actions/events";
+
+const store = configureStore();
+
+store.dispatch(addEvent({ title: "new event", date: 123456, id: "myid" }));
+console.log(store.getState());
+store.dispatch(editEvent("myid", { title: "changed title" }));
+console.log(store.getState());
+store.dispatch(removeEvent({ id: "myid" }));
+console.log(store.getState());
```

Comment:
Do you need that mocked dispatching actions? If so, then maybe add an appropriate comment at the beginning of that. I.e.:
`// dispatch mocked data for development purposes` or something like that.